.circleci/config.yml

```diff
@@ -486,6 +491,26 @@ jobs:
           file: ./merged.xml
           flags: phpunit
 
+  pa11y-ci:
+    machine:
+      image: circleci/classic:latest
+    steps:
+      - checkout
+      - attach_workspace: { at: ~/project }
+      - run:
+          name: Set environment
+          command: |
+            ~/project/.circleci/set_env.sh >> $BASH_ENV
+      - run:
+          name: Run pa11y
+          command: |
+            docker-compose -f docker-compose.yml up -d pa11y
+            docker-compose exec frontend touch /var/www/.enableProdMode
+            docker-compose exec admin touch /var/www/.enableProdMode
+            docker-compose exec api touch /var/www/.enableProdMode
+            sleep 10
```

Comment:
I'd tend to avoid sleep and prefer something that waits for a service to be ready.

Reply:
Good point actually - we're using `waitforit` in Serve-OPG if you want to see how that would work @Gugandeep https://github.com/ministryofjustice/serve-opg/blob/master/.circleci/config.yml

Reply:
Added a ticket to the backlog to implement this: https://opgtransform.atlassian.net/browse/DDPB-3518